### PR TITLE
enable authorization based on AuthToken

### PR DESCRIPTION
**Pull request by gsturov**

### CODE REVIEW

File: car_framework/car_service.py [hunk 1]
```diff
@@ -113,38 +112,24 @@ def check_import_status(self, statuses):
...
-        
+
 
     def delete(self, resource, ids):
         # report and source not mentioned anywhere coz connectors aren't allowed to delete it
         key_based = ["ipaddress", "hostname", "macaddress"]
         external_id_based = ["asset", "container", "user", "account", "application", "database", "port", "vulnerability", "geolocation"]
-        
         if resource in key_based:
-            resource_key = 'keys'
```
Comment (baulus) [inline car_framework/car_service.py:124]: Seems you overrode my changes in delete function:
https://github.com/IBM/cp4s-car-connector-framework/pull/5/files#diff-213769a12cf7317bbd297d6d5d37ffed47b1f02ab8dc826bfc789fcac85a8916L123
was there any specific reason for that or it is merge problem?

File: car_framework/car_service.py [hunk 2]
```diff
@@ -272,32 +257,17 @@ def wait_until_done(self, job_id):
...
 
     def exit_full_import_in_progress_state(self):
-        endpoint = '%s/source/%s%s' % (self.car_url, context().args.source, FULL_IMPORT_IN_PROGRESS_ENDPOINT)
+        endpoint = 'source/%s%s' % (context().args.source, FULL_IMPORT_IN_PROGRESS_ENDPOINT)
         r = self.communicator.delete(endpoint)
         job_id = get_json(r)['job_id']
         self.wait_until_done(job_id)
         return r.status_code
 
-    def compose_paginated_list(self, ids):
```
Comment (baulus) [inline car_framework/car_service.py:289]: the same as in previous comment

File: car_framework/car_service.py (hunk 1 above)
Comment (gsturov) [reply car_framework/car_service.py:124]: Right, merge problem. Let me fix it...

Comment (gsturov) [reply car_framework/car_service.py:124]: Fixed

File: car_framework/car_service.py (hunk 2 above)
Comment (gsturov) [reply car_framework/car_service.py]: Fixes

**Review by baulus — APPROVED** [review]

**Pull request merged**